specimenwindow.py:92: GtkWarning: GtkSpinButton: setting an adjustment 
with non-zero page size is deprecated

--- origsrc/gnome-specimen-0.4/data/gnome-specimen.glade	2007-12-01 13:18:06.000000000 -0600
+++ src/gnome-specimen-0.4/data/gnome-specimen.glade	2010-05-10 02:08:06.306693200 -0500
@@ -255,7 +255,7 @@
                         <property name="tooltip" translatable="yes">Font size used for the previews</property>
                         <property name="max_length">3</property>
                         <property name="width_chars">3</property>
-                        <property name="adjustment">16 1 144 1 10 10</property>
+                        <property name="adjustment">16 1 144 1 10 0</property>
                         <property name="climb_rate">1</property>
                         <property name="numeric">True</property>
                         <property name="update_policy">GTK_UPDATE_IF_VALID</property>
--- origsrc/gnome-specimen-0.4/specimen/main.py	2007-06-05 03:26:30.000000000 -0500
+++ src/gnome-specimen-0.4/specimen/main.py	2010-05-10 02:10:03.615402900 -0500
@@ -39,11 +39,6 @@ def main(args):
         exit_with_error('Importing gtk.glade module failed',  e)
 
     try:
-        import gnome
-    except ImportError, e:
-        exit_with_error('Importing gnome module failed',  e)
-
-    try:
         import gconf
     except ImportError, e:
         exit_with_error('Importing gconf module failed',  e)
@@ -56,8 +51,6 @@ def main(args):
     gtk.glade.bindtextdomain(config.PACKAGE, config.LOCALEDIR)
     gtk.glade.textdomain(config.PACKAGE)
 
-    prog = gnome.program_init(config.PACKAGE, config.VERSION)
-
     gtk.window_set_default_icon_name('gnome-specimen')
 
     from specimenwindow import SpecimenWindow
